src/lib/bootloader/grub2efi.rb

```diff
@@ -109,7 +113,8 @@ def packages
 
     # overwrite BootloaderBase version to save secure boot
```

Comment:
in fact no, as difference between grub2 and grub2efi version is only secure boot, now when both have trusted boot option.